extractor/filesystem/os/pacman/pacman.go

```diff
@@ -95,9 +94,12 @@ func (e Extractor) Requirements() *plugin.Capabilities { return &plugin.Capabili
 
 // FileRequired returns true if the specified file matches the "desc" file patterns.
 func (e Extractor) FileRequired(api filesystem.FileAPI) bool {
+	// archPrefix and archSuffix are used to match the right file and location.
+	archPrefix := "var/lib/pacman/local"
```

Comment:
This should have a / at the end - otherwise it'll try to extract a file like `/var/lib/pacman/localfoo/desc`

Reply:
Let's also add a test case for this